src/components/__tests__/chip.spec.jsx

```diff
@@ -23,4 +23,14 @@ describe('Chip component', () => {
     const removeIcon = queryByTestId('remove-icon');
     expect(removeIcon).toBeNull();
   });
+
+  it('should not have disabled class when disabled=false passed as a prop', () => {
+    const { container } = render(<Chip disabled={false}>Some content</Chip>);
+    expect(container.firstChild.className).not.toMatch(/disabled/);
```

Comment:
I would probably prefer to test the actual `disabled` attribute instead of the class name as it's more representative of what the browser will allow the user to do.
And, to make it more robust to possible implementation changes, I would go for a testing-library query (...ByTestID, ...ByRole) instead of getting the firstChild.
Something like
```js
expect(screen.getByRole('button')).toHaveAttribute('disabled')
```

Reply:
Updated